Dockerfile.dapper

```diff
@@ -22,7 +22,6 @@ RUN if [ "$(go env GOARCH)" = "arm64" ]; then
     tar -zxvf trivy_0.11.0_Linux-64bit.tar.gz                                                            && \
     mv trivy /usr/local/bin;                                                                                \
     fi
-RUN trivy --download-db-only
```

Comment:
this removal confuses me: why do we need the gh auth token then?

Reply:
This step is useless because caching it here is useless for a couple reasons:
* Whatever we download and cache here will likely be expired in most of the runs that use this cached image
* Whatever we download here is in reality unused because we mount a cache volume over the top of the directory that this gets downloaded to